vault/core.go

```diff
@@ -757,6 +760,9 @@ type CoreConfig struct {
 	// Use the deadlocks library to detect deadlocks
 	DetectDeadlocks string
 
+	// Disables tracking a leases role
```

Comment:
I think it's worth clarifying here that this is if there is no role-based lease-count quota, not just in general

Reply:
Actually, let me do this as a suggestion :)